packages/block-editor/src/components/link-control/style.scss

```diff
@@ -192,6 +192,7 @@ $preview-image-height: 140px;
 		flex-direction: row;
 		align-items: flex-start;
 		margin-right: $grid-unit-10;
+		min-width: 0;
```

Comment:
I think we should include a comment to explain what this does and why it's necessary. That would help avoid regressions.

Reply:
👍 

I can add some comments later today (unless the alternative PR #36993 is merged).